### PR TITLE
Compile target also for macOS

### PR DESCRIPTION
**Pull request by marcoconti83**

### CODE REVIEW

File: Source/Public/ZMTBaseTest.m [hunk 1]
```diff
@@ -19,7 +19,9 @@
 
 @import WireSystem;
 @import OCMock;
-@import MobileCoreServices;
+#if TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR
+    @import MobileCoreServices;
+#endif
```
Comment (mikeger) [inline Source/Public/ZMTBaseTest.m]: So it works automatically on Mac, i.e. the APIs we are importing from `MobileCoreServices` are available without the import on the mac?

Comment (marcoconti83) [inline Source/Public/ZMTBaseTest.m]: It compiles fine on mac

Comment (mikeger) [inline Source/Public/ZMTBaseTest.m]: Ok, may be it would compile for iOS without the import, too?

**Comment by marcoconti83** [comment]
It does! Removed

**Pull request merged**